src/StreamJsonRpc.Desktop/Resources.resx

```diff
@@ -179,6 +187,10 @@
   <data name="NonNegativeIntegerRequired" xml:space="preserve">
     <value>A non-negative integer is required.</value>
   </data>
+  <data name="OverloadsMissingMethodAttribute" xml:space="preserve">
+    <value>The {0} method has {1} defined, but not all of its overloads do.</value>
```

Comment:
I think you can delete the message and reuse the `ConflictingMethodNameAttribute` message after you reword that as I have suggested.